agenta-backend/agenta_backend/services/event_db_manager.py

```diff
@@ -85,7 +85,7 @@ async def create_app_trace(payload: CreateTrace, **kwargs: dict) -> str:
     return trace_db_to_pydantic(trace)["trace_id"]
 
 
-async def get_single_trace(trace_id: str, **kwargs: dict) -> Trace:
+async def get_single_trace_object(trace_id: str, **kwargs: dict) -> Trace:
```

Comment:
We use "_object" when returning the db model (TraceDB). In this case, it should be just `get_single_trace`.

Reply:
This would conflict with this https://github.com/devgenix/agenta/blob/438933f0ffecba41abd4582fd782cc4c41a8f3fe/agenta-backend/agenta_backend/routers/observability_router.py#L71C2-L71C2

Resolved it another way by swapping the function name convention.